prisma/seed.ts

```diff
@@ -36,25 +37,40 @@ function throwIfNotLocalhost() {
   );
 }
 
-async function seedNewData() {
-  await seedUsers();
-  await prisma.plusStatus.createMany({ data: getPlusStatusesData() });
-  await prisma.plusSuggestion.createMany({ data: getPlusSuggestionsData() });
-  await prisma.plusVotingSummary.createMany({
-    data: getPlusVotingSummaryData(),
-  });
+async function dropAllData() {
```

Comment:
The additional challenge here is that everything has to be deleted in the right order otherwise Postgres complains. I think the easiest way for now is just to change the order of these as needed.

Also maybe worth it to see the discussion here for alternative solutions: https://github.com/prisma/docs/issues/451